scout/adapter/mongo/clinvar.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+import logging
+from datetime import datetime
+from scout.exceptions import IntegrityError
+
+import pymongo
+
+LOG = logging.getLogger(__name__)
+
+class ClinVarHandler(object):
+    """Class to handle clinvar submissions for the mongo adapter"""
+
+    def add_clinvar_submission(self, submission_obj, user, institute, case_id):
+        """Add a clinvar submission for a case. A submission consists of one or more variants added to the clinvar collection.
+
+            Args:
+                submission_obj(str)
```

Comment:
The submission object is not a string, please change and describe more.
Is is just a dictionary with variants or is there any extra information?